src/main/scala/ru/d4team/chat/models/person/PersonRecord.scala

```diff
@@ -0,0 +1,17 @@
+package ru.d4team.chat.models.person
+
+import io.scalaland.chimney.dsl.TransformerOps
+
+import java.time.Instant
+import java.util.UUID
+
+final case class PersonRecord(
```

Comment:
Why do we need 2 additional Identical classes for person?

Reply:
It is the best practice, since it reduces interconnectivity between levels and in the future we will able to add changes for particular level model without any problems

Reply:
Ok, due to there isn't some service it was not obvious for me)